test/test_nexus_parser.rb

```diff
@@ -754,6 +754,28 @@ def test_strange_chr_state_labels
 
   end
 
+  # https://github.com/mjy/nexus_parser/issues/9
+  def test_three_both_numeric_and_label_state_names_in_a_row
+    input =" CHARSTATELABELS
+    1 'Metatarsal trichobothria (CodAra.29)' / 3 9 27 asdf;
```

Comment:
Thanks for the review, I was wondering that too - the error occurs with any triple of the form Number, (both Label and Number), (both Label and Number), so it's not so much the format of the numbers that matters in terms of testing for regression (aside from the second two also being Labels), maybe a comment would be in order. That said I think this will go away with whatever changes are made in issue 16.

Reply:
Going to merge this, can be tweaked from there.